schema/va-spec/aac-2017/diagnostic-study-statement-profile-source.yaml

```
@@ -28,3 +28,50 @@ $defs:
             A proposition about a diagnostic association between a variant and condition, for which the study provides
             evidence. The validity of this proposition, and the level of confidence/evidence supporting it,
             may be assessed and reported by the Statement.
+        strength:
+          description: >-
+            The strength of support that the Statement is determined to provide for or against the Diagnostic Proposition
+            for the assessed variant, based on the curation and reporting conventions of the AMP/ASCO/CAP (AAC) 2017
+            Guidelines. Note that the enumerated value set here is bound to the ‘code’ field of the Coding object that
+            is nested inside a MappableConcept's primary coding. Conditional requirement: Strength is evaluated relative to
+            the direction indicated by the 'direction' attribute. If direction is either 'supports' or 'disputes', the
+            'strength' attribute is required.  If the direction is 'neutral', then the 'strength' attribute is not allowed.
+          properties:
+            primaryCoding:
+              code:
+                enum:              # TO DO: Kori/Alex to populate value set here, as appropriate
+                  -                
+                  -  
+                  -
+              system:
+                const: AMP/ASCO/CAP (AAC) Guidleines for the Interpretation and Reporting of Sequence Variants in Cancer, 2017
+        classification:
+          description: >-
+            A single term or phrase classifying the subject variant based on the outcome of direction and strength assessments of
+            the Statement's Proposition - reported here using terms from the AMP/ASCO/CAP (AAC) 2017 Guidelines. Note that the
+            enumerated value set here is bound to the ‘code’ field of the Coding object that is nested inside a MappableConcept's
+            primary coding.
+          properties:
+            primaryCoding:
+              code:
+                enum:              # TO DO: Kori/Alex to populate value set here, as approriate
+                  -                
+                  - 
+                  -  
+              system:
+                const: AMP/ASCO/CAP (AAC) Guidleines for the Interpretation and Reporting of Sequence Variants in Cancer, 2017
```

This follows ClinVar: https://github.com/ncbi/clinvar/blob/master/submission_api_schema/submission_api_schema.json#L510-L514

```suggestion
                enum:
                  - "Tier I - Strong"                
                  - "Tier II - Potential"
                  -  "Tier III - Unknown"
                  - "Tier IV - Benign/Likely benign"
              system:
                const: AMP/ASCO/CAP (AAC) Guidleines for the Interpretation and Reporting of Sequence Variants in Cancer, 2017
```